netutils/esp8266/esp8266.c

```diff
@@ -2478,7 +2478,8 @@ ssize_t lesp_send(int sockfd, FAR const uint8_t *buf, size_t len, int flags)
 
   if (ret >= 0)
     {
-      ret = lesp_ask_ans_ok(lespTIMEOUT_MS, "AT+CIPSEND=%d,%d\r\n", sockfd, len);
+      ret = lesp_ask_ans_ok(LESP_TIMEOUT_MS,
+                            "AT+CIPSEND=%d,%d\r\n", sockfd, len);
```

Comment:
Is this an option here?
```suggestion
      ret = lesp_ask_ans_ok(LESP_TIMEOUT_MS, "AT+CIPSEND=%d,%d\r\n",
                            sockfd, len);
```